apps/climatemappedafrica/src/pages/api/hurumap/profiles.js

```diff
@@ -0,0 +1,15 @@
+import { fetchProfile } from "@/climatemappedafrica/lib/hurumap";
+
+export default async function index(req, res) {
+  if (req.method === "GET") {
+    try {
+      const { geoCode } = req.query;
+      const result = await fetchProfile(geoCode);
+      return res.status(200).json(result);
+    } catch (err) {
+      return res.status(500).json(err.message);
+    }
```

Comment:
🤔 ... shouldn't `404` be thrown when we have invalid `geoCode`? `500` means server error (which can happen but not always).